### PR TITLE
OBPIH-2744 Dashboard should refresh on location change

### PR DESCRIPTION
**Pull request by bamoure**

### CODE REVIEW

**Comment by jmiranda** [comment]
@awalkowiak Could you guys review this since it's mostly React?

File: src/js/reducers/indicatorsReducer.jsx [hunk 1]
```diff
@@ -46,6 +48,10 @@ export default function (state = initialState, action) {
         data: newState,
       };
     }
+    case RESET_INDICATORS:
+      return {
```
Comment (awalkowiak) [inline src/js/reducers/indicatorsReducer.jsx]: You could return initialState for clarity.

File: src/js/components/tablero/Tablero.jsx [hunk 2]
```diff
@@ -77,18 +82,17 @@ class Tablero extends Component {
   componentDidMount() {
     this.fetchData();
   }
+  componentWillReceiveProps(nextProps) {
```
Comment (awalkowiak) [inline src/js/components/tablero/Tablero.jsx]: There should be empty line between functions

Comment (awalkowiak) [reply src/js/components/tablero/Tablero.jsx]: Also why there is no constructor in this component and state had no numberData, which were updated and used before?

File: src/js/reducers/indicatorsReducer.jsx [hunk 3]
```diff
@@ -65,6 +71,11 @@ export default function (state = initialState, action) {
         ...state,
         data: arrayArchive(state.data, action.payload.index),
       };
+    case FETCH_NUMBERS:
+      return {
+        ...state,
+        numberData: action.payload.data,
```
Comment (awalkowiak) [inline src/js/reducers/indicatorsReducer.jsx]: Why there is no numberData in initial state?

File: src/js/components/tablero/Tablero.jsx (hunk 2 above)
Comment (bamoure) [reply src/js/components/tablero/Tablero.jsx]: Constructor added. numberData was used in the state before, but is now mapped in props.

Comment (awalkowiak) [reply src/js/components/tablero/Tablero.jsx]: I meant that it was not initialized. But it is ok now :)

**Comment by jmiranda** [comment]
@awalkowiak Can you review this one again?

**Pull request merged**